main.py

```diff
@@ -146,6 +150,11 @@ def __init__(self, *args, obj=None, **kwargs):
         config_register_callback(self.error_message_box)
         packet_init()
 
+        # Disable UART config if Linux TODO: Make this compatible, tty etc as well as COM
+        if OS == "Linux":
```

Comment:
I think this should be Raspberry Pi?

Reply:
This can be deleted as there is a RaspberryPi check on line 136

Reply:
ok, can I have a look at it running on RaspPi and Windows?

Reply:
Demo performed, all ok.